cranelift/codegen/src/isa/x64/inst.isle

```diff
@@ -1057,7 +1057,10 @@
 ;;
 ;; This is used when lowering various shifts and rotates.
 (decl put_masked_in_imm8_gpr (Value Type) Imm8Gpr)
-(extern constructor put_masked_in_imm8_gpr put_masked_in_imm8_gpr)
+(rule 1 (put_masked_in_imm8_gpr (u64_from_iconst amt) ty)
```

Comment:
Was this priority necessary for correct output? I think it should be fine to omit it.

Reply:
Not really, but I've had issues with priorities, so now I'm a bit paranoid about it. I'll remove it